common/config/rush/pnpm-lock.yaml

```diff
@@ -57,18 +57,18 @@ dependencies:
   '@types/tough-cookie': 2.3.5
   '@types/tunnel': 0.0.1
   '@types/uuid': 3.4.5
-  '@types/webpack': 4.39.0
```

Comment:
Was this file updated by running `rush update` or `rush update --full`?  It appears to be the latter, since unrelated dependencies are being updated.  Can you please revert this file and run `rush update` to just add/update the dependencies modified in this PR?